.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
```

Comment:
Use some linting tool as well. Refer [this](https://www.sitepoint.com/comparison-javascript-linting-tools/) for comparing different tools. For setting it up on Travis CI, refer [this](https://github.com/dwyl/learn-travis/blob/master/README.md)

Reply:
I'll set up ESlint in a separate PR

Reply:
Alright, I'm creating an issue for setting up ESLint, resolve that :+1: